generate_pages.py

```diff
@@ -0,0 +1,61 @@
+import argparse
+import os
+import yaml
+from github import Github
```

Comment:
There's no such library in the Pipfile. This will just crash the interpreter.

Please refer to the documentation of Pipenv on how to add dependencies to your project.